main.go

```diff
@@ -37,6 +37,7 @@ var (
 	timeGranularity = flag.String("granularity", "day", "how often we expect to append new data")
 	streamStart     = flag.String("streamStart", "", "The start of the streamed data. Only used if granularity='stream'. Used to ensure idempotency")
 	streamEnd       = flag.String("streamEnd", "", "The end of the streamed data. Only used if granularity='stream'. Used to ensure idempotency")
+	isTargetPT      = flag.Bool("targetPT", false, "Whether or not the target data time column is in PT.")
```

Comment:
I should have thought of this before, but what about if we just pass in the timezone "America/Los_Angeles" as a param, defaulting to "UTC"?